### PR TITLE
Migration to Catberry 9

### PR DESCRIPTION
**Pull request by reenko**

### CODE REVIEW

**Comment by pragmader** [comment]
This PR can be merged only after Catberry 9 is released. Therefore current builds are failing which is okay.

File: .gitignore [hunk 1]
```diff
@@ -12,3 +12,4 @@ Thumbs.db
 __BrowserBundle.js
```
Comment (kalyanov) [inline .gitignore]: This is redundant now

File: build.js [hunk 2]
```diff
@@ -4,20 +4,12 @@ const IS_RELEASE = process.argv.length === 3 ?
...
-		useImageOptimization: false,
-		useSvgOptimization: false,
-		useSvgSymbols: false,
-		banner: '/**\n * Catberry Homepage styles\n * <%now%>\n */\n',
-		useNormalizeCss: true
-	}
-});
+const config = configLoader.load();
+const cat = catberry.create(config);
+config.isRelease = IS_RELEASE === undefined ? config.isRelease : IS_RELEASE;
```
Comment (kalyanov) [inline build.js]: I think config.isRelease should be set before catberry.create

File: catberry_components/head/index.js [hunk 3]
```diff
@@ -11,6 +11,21 @@ const ComponentBase = require('../../lib/ComponentBase');
 class Head extends ComponentBase {
 
 	/**
+	 * Creates new instance of the "head" component.
+	 * @param {ServiceLocator} locator.
```
Comment (kalyanov) [inline catberry_components/head/index.js]: Let's add a descripton to locator
The service locator for resolving dependencies.

File: catberry_components/head/index.js [hunk 4]
```diff
@@ -28,6 +43,8 @@ class Head extends ComponentBase {
 			.then((pages) => this.localizeContext({
```
Comment (kalyanov) [inline catberry_components/head/index.js]: Braces is not required here

File: configs_example/basic.json [hunk 5]
```diff
@@ -1,4 +1,6 @@
 {
```
Comment (kalyanov) [inline configs_example/basic.json]: Maybe we have to put template configs in config_template folder to avoid any confusions.

Comment (pragmader) [inline configs_example/basic.json]: I'm not sure if I got what you mean. Which configs?

File: catberry_components/head/index.js (hunk 3 above)
Comment (pragmader) [inline catberry_components/head/index.js]: Good to have but not necessary.

File: catberry_components/head/index.js (hunk 4 above)
Comment (reenko) [inline catberry_components/head/index.js]: In this repo always single param have braces.

File: configs_example/basic.json (hunk 5 above)
Comment (kalyanov) [inline configs_example/basic.json]: basic.json, browser.json and environment.json, with {{PARAM}}

catberry_example folder seems like folder with some example files, not files for configs building. It's not obvious.

Probably we have to rename folder configs_example to configs_template.

File: catberry_components/head/index.js (hunk 4 above)
Comment (kalyanov) [inline catberry_components/head/index.js]: ок

File: configs_example/basic.json (hunk 5 above)
Comment (pragmader) [inline configs_example/basic.json]: Agree, but it also could be out of this PR's scope.

Comment (kalyanov) [inline configs_example/basic.json]: Add an issue https://github.com/catberry/catberry-homepage/issues/37

**Pull request merged**